apps/system-e2e/src/tests/judicial-system/utils/helpers.ts

```diff
@@ -30,19 +30,20 @@ async function createFakePdf(title: string) {
   }
 }
 
-export async function chooseDocument(
+export const chooseDocument = async (
   page: Page,
   clickButton: () => Promise<void>,
   fileName: string,
-) {
+) => {
   const fileChooserPromise = page.waitForEvent('filechooser')
   await clickButton()
 
   const fileChooser = await fileChooserPromise
-  await fileChooser.setFiles(await createFakePdf(fileName))
+  await page.waitForTimeout(1000)
```

Comment:
Why wait some time (which slows tests down), when you could be waiting for some _thing_

Reply:
The reason for this is explained in the PR description. This wait was in the initial file selection test, but later removed because we thought we had found the problem. That, however, turned out to be wrong so we are adding it back. We will continue to improve, but we want and need the tests to pass to be useful. So, we will lieve with this for now.

Reply:
An unfortunate solution, but we must go on and get stuff done :muscle: